.github/workflows/sdks.yml

```diff
@@ -50,17 +58,21 @@ jobs:
             SWIFT_BRANCH="development"
           fi
           cd
-          SWIFT_TAG="${{ steps.check.outputs.latest-tag }}"
-          wget -q https://download.swift.org/$SWIFT_BRANCH/ubuntu2204/$SWIFT_TAG/$SWIFT_TAG-ubuntu22.04.tar.gz
+          if [[ ${{ matrix.os }} = 'macos'* ]]; then
```

Comment:
You can't check `matrix.os` here as it's not defined for this job. That's why I said to check `steps.cache-toolchain-macos.outputs.cache-hit` instead. 😉

Reply:
Oops! Fixed in https://github.com/finagolfin/swift-android-sdk/pull/162/commits/154f77712a020df0958f2fa2776e80f55033330e

Reply:
Hmm, I don't know why it is failing now...

Reply:
I think I just missed quoting the `${{ steps.cache-toolchain-macos.outputs.cache-hit }}` check in the shell script (I didn't test the cache miss scenario in my fork's CI). I think it should be fixed with https://github.com/finagolfin/swift-android-sdk/pull/162/commits/58399f4bfa0da5e8a2f625c4b2824ab825c624a8 if you give it another run…

Reply:
Yep, that part works now. 👍